content/concepts/module-resolution.md

```diff
@@ -47,7 +47,7 @@ require("module/lib/file");
 ```
 
 Modules are searched for inside directories which are specified using `resolve.modules`, which is can be an array comprising of different paths.
-Aliasing, ie setting `resolve.alias` to an existing module path, allows you to replace the module path with an alias name during `require/import`.
+Aliasing, i. e. setting `resolve.alias` to an existing module path, allows you to replace the module path with an alias name during `require/import`.
```

Comment:
This would be good to add to the writer's guide. I mean i.e. vs. i. e. . Ideally that should be linted.